src/helpers/getIPFSContent.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+import env from '@/helpers/env'
+
+export default async function (cid?: string) {
+  if (!cid) return
+  const ipfsBaseUrl = env.IPFS_GATEWAY
+  const ipfsUrlString = `${ipfsBaseUrl}/${cid}`
+  const ipfsContent = (await axios.get<{ text?: string }>(ipfsUrlString)).data
```

Comment:
How long will it wait before canceling the request? I remember `fetch` waited for too long, we need something like 10-15s

Reply:
Do you think it can take more than 1-2 seconds?

Reply:
5 seconds